### PR TITLE
[id] add safe sysctl

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/29841/files
/assign @habibrosyad

### CODE REVIEW

File: content/id/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Sysctl berikut ini didukung dalam kelompok _safe_:
 - `net.ipv4.ip_local_port_range`,
 - `net.ipv4.tcp_syncookies`,
 - `net.ipv4.ping_group_range` (sejak Kubernetes 1.18).
```
Comment (jihoon-seo) [inline content/id/docs/tasks/administer-cluster/sysctl-cluster.md:58]: Optional nit:
(I can see that the period still exists in EN page, but for consistency between unordered list elements:)
(We can also replace this period with comma in EN page:)

```suggestion
- `net.ipv4.ping_group_range` (sejak Kubernetes 1.18),
```

**Review by jihoon-seo — COMMENTED** [review]
/lgtm

File: content/id/docs/tasks/administer-cluster/sysctl-cluster.md (hunk 1 above)
Comment (Arhell) [reply content/id/docs/tasks/administer-cluster/sysctl-cluster.md]: done
https://github.com/kubernetes/website/pull/31614

**Comment by habibrosyad** [comment]
Thanks
/lgtm
/approve

I wonder why bot assigned so many reviewers.

**Pull request merged**